src/g_game.c

```diff
@@ -4616,37 +4615,37 @@ void G_BindCompVariables(void)
 #define BIND_EMU(id, v, help) \
   M_BindBool(#id, &overflow[(id)].enabled, NULL, (v), ss_none, wad_no, help)
 
-  BIND_EMU(emu_spechits, true, "SPECHITS overflow emulation");
-  BIND_EMU(emu_reject, true, "REJECT overflow emulation");
+  BIND_EMU(emu_spechits, true, "SPECHITS-overflow emulation");
+  BIND_EMU(emu_reject, true, "REJECT-overflow emulation");
   M_BindBool("emu_intercepts", &overflow[emu_intercepts].enabled, NULL, true,
-    ss_comp, wad_no, "INTERCEPTS overflow emulation");
-  BIND_EMU(emu_missedbackside, false, "Missed backside emulation");
-  BIND_EMU(emu_donut, true, "Donut overrun emulation");
+    ss_comp, wad_no, "INTERCEPTS-overflow emulation");
+  BIND_EMU(emu_missedbackside, false, "Missed-backside emulation");
+  BIND_EMU(emu_donut, true, "Donut-overrun emulation");
 }
 
 void G_BindWeapVariables(void)
 {
   M_BindNum("view_bobbing_pct", &view_bobbing_pct, NULL, 4, 0, 4, ss_weap, wad_no,
-            "Player View Bobbing (0 - 0%, 1 - 25% ... 4 - 100%)");
+            "Player view bobbing (0 = 0%; 1 = 25%; ... 4 = 100%)");
   M_BindNum("weapon_bobbing_pct", &weapon_bobbing_pct, NULL,
             4, 0, 4, ss_weap, wad_no,
-            "Player Weapon Bobbing (0 - 0%, 1 - 25% ... 4 - 100%)");
+            "Player weapon bobbing (0 = 0%; 1 = 25%; ... 4 = 100%)");
   M_BindBool("hide_weapon", &hide_weapon, NULL, false, ss_weap, wad_no,
              "1 to hide weapon");
   M_BindNum("center_weapon", &center_weapon, NULL, 0, 0, 2, ss_weap, wad_no,
-            "1 to center the weapon sprite during attack, 2 to keep it bobbing");
+            "Weapon alignment while attacking (1 = Centered; 2 = Bobbing)");
   M_BindBool("weapon_recoilpitch", &weapon_recoilpitch, NULL,
              false, ss_weap, wad_no,
              "1 to enable recoil pitch from weapon fire");
 
   M_BindBool("weapon_recoil", &default_weapon_recoil, &weapon_recoil,
              false, ss_none, wad_yes,
-             "1 to enable recoil from weapon fire (Boom version)");
+             "1 to enable physical recoil from weapon fire");
```

Comment:
We have several recoil and bobbing options, the difference is that the new options are purely cosmetic and do not affect compatibility. The old `weapon_recoil` and `player_bobbing` are for backwards compatibility only and are not recommended. Not sure how to indicate this.

Reply:
I'm aware that the new options are just cosmetic, which is why I refer to these as "physical". Perhaps we can add something like `"(demo-unsafe)"` to their descriptions?

Reply:
Technically it's demo compatible but for complevels boom, mbf, mbf21. For example here is mbf21 demo with recoil that works on DSDA-Doom: [recoil.zip](https://github.com/fabiangreffrath/woof/files/15243521/recoil.zip)

I don't like "physical". Maybe something like "(affects compatibility)".

Reply:
> I don't like "physical". Maybe something like "(affects compatibility)".

I insist on keeping "physical" because it differentiates these features from the cosmetic ones from a gameplay point of view, compatibility aside. "(affects compatibility)" sounds good to me; I would simply append that to the existing descriptions.

Reply:
> I insist on keeping "physical" because it differentiates these features from the cosmetic ones from a gameplay point of view, compatibility aside. "(affects compatibility)" sounds good to me; I would simply append that to the existing descriptions.

Okay, sounds good to me.